src/pages/hotel/booking/booking-confirmation/hotel-overview/HotelOverview.scss

```diff
@@ -23,6 +23,7 @@
 .hotel-name-book {
   font-size: 1.3rem;
   margin-top: 10px;
+  margin-left:40px;
```

Comment:
This will results to increase margins between Hotel image and details all the cases. Instead of directly increasing the margin for all, may be we can think of other solutions like maintaining a min-width for Hotel image, etc.